docs/en/sql-reference/window-functions/last_value.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: /en/sql-reference/window-functions/last_value
+sidebar_label: last_value
+sidebar_position: 4
+---
+
+# last_value
+
+Returns the last non-NULL value evaluated within its ordered frame.
```

Comment:
Same as first_value. This supports `RESPECT NULLS`

Reply:
@Algunenano thanks, have pushed the changes.